cookie-clicker-lab4-7/cookie-clicker-lab4-7/src/main/java/edu/csula/web/AuthenticationServlet.java

```diff
@@ -0,0 +1,91 @@
+package edu.csula.web;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.Collection;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/admin/auth")
+public class AuthenticationServlet extends HttpServlet {
+	@Override
+	public void doGet( HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		// TODO: render the authentication page HTML
+		response.setContentType("text/html");
+		PrintWriter out = response.getWriter();
+		String html;
+		html = "<form action="LoginServlet" method="post">";  
```

Comment:
This action is leading the login to send request to somewhere else.

In other word, the login is not working because of this.